### ISSUE TITLE
[DistSQL] Add some new syntax to manage scaling configurations.

### ISSUE THREAD

**Issue by RaigorJiang** [apache/shardingsphere#14661]
Hi community,

In order to facilitate users to manage the sharding scaling rule configuration, we will add some new DistSQL syntax.
At the same time, since the scaling feature has been refactored to support different levels of configurations (#13875), DistSQL will also support different levels:

#### Example
- [x] 1. Automatic scaling process, specifying minimal rule configuration
```sql
CREATE SHARDING SCALING RULE scaling_name(
COMPLETION_DETECTOR(TYPE(NAME=IDLE, PROPERTIES("incremental-task-idle-minute-threshold"=30))),
DATA_CONSISTENCY_CHECKER(TYPE(NAME=DATA_MATCH, PROPERTIES("chunk-size"=1000)))
);
```
- [x] 2. Automatic scaling process, specifying complete rule configuration
```sql
CREATE SHARDING SCALING RULE scaling_name(
INPUT(
  WORKER_THREAD=40,
  BATCH_SIZE=1000,
  RATE_LIMITER(TYPE(NAME=QPS, PROPERTIES("qps"=50)))
),
OUTPUT(
  WORKER_THREAD=40,
  BATCH_SIZE=1000,
  RATE_LIMITER(TYPE(NAME=TPS, PROPERTIES("tps"=2000)))
),
STREAM_CHANNEL(TYPE(NAME=MEMORY, PROPERTIES("block-queue-size"=10000))),
COMPLETION_DETECTOR(TYPE(NAME=IDLE, PROPERTIES("incremental-task-idle-minute-threshold"=30))),
DATA_CONSISTENCY_CHECKER(TYPE(NAME=DATA_MATCH, PROPERTIES("chunk-size"=1000)))
);
```
- [x] 3. Manual scaling process
```sql
CREATE SHARDING SCALING RULE scaling_name(
INPUT(
  WORKER_THREAD=40,
  BATCH_SIZE=1000,
  RATE_LIMITER(TYPE(NAME=QPS, PROPERTIES("qps"=50)))
),
OUTPUT(
  WORKER_THREAD=40,
  BATCH_SIZE=1000,
  RATE_LIMITER(TYPE(NAME=TPS, PROPERTIES("tps"=2000)))
),
STREAM_CHANNEL(TYPE(NAME=MEMORY, PROPERTIES("block-queue-size"=10000)))
);
```
- [x] 4. Create an empty rule configuration (done in #14551)
```sql
CREATE SHARDING SCALING RULE scaling_name;
```
- [x] 5. Drop sharding scaling rule configuration (done in #14551)
```sql
DROP SHARDING SCALING RULE scaling_name;
```
- [x] 6. Enable sharding scaling rule configuration
```sql
ENABLE SHARDING SCALING RULE scaling_name;
```
- [x] 7. Disable sharding scaling rule configuration
```sql
DISABLE SHARDING SCALING RULE scaling_name;
```
- [x] 8. Show sharding  scaling rules
```sql
SHOW SHARDING SCALING RULES [from schemaName];
```

**Comment by sandynz** [comment]
The design and implementation cost a short time, since scaling require DistSQL in `5.1.0` release recently. Thank you very much! @RaigorJiang 

I've tested all the `CREATE` examples above, every one of them works. I just did more test now, and I found some situation:

1. Looks there're strict constraints.

Empty scaling configuration is allowed, it means `INPUT`, `OUTPUT`, `STREAM_CHANNEL`, `COMPLETION_DETECTOR` and `DATA_CONSISTENCY_CHECKER` are optional.

Seems they're not optional for now, in `minimal automatic process`, `complete automatic process` and `manual process`.
When any of them are not configured, it show error message. e.g.

```
mysql> CREATE SHARDING SCALING scaling_name1(
    -> COMPLETION_DETECTOR(TYPE(NAME=IDLE, PROPERTIES("incremental-task-idle-minute-threshold"=30)))
    -> );
ERROR 1235 (42000): This version of ShardingSphere-Proxy doesn't yet support this SQL. 'You have an error in your SQL syntax'
```
